node-stringprep.cc

```diff
@@ -43,11 +43,9 @@ class StringPrep : public ObjectWrap {
 
   static NAN_METHOD(New)
   {
-    NanScope();
-
-    if (args.Length() >= 1 && args[0]->IsString())
+    if (info.Length() >= 1 && info[0]->IsString())
       {
-        String::Utf8Value arg0(args[0]->ToString());
+        Nan::Utf8Value arg0(info[0]->ToString());
```

Comment:
../node-stringprep.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE StringPrep::New(Nan::NAN_METHOD_ARGS_TYPE)’:

../node-stringprep.cc:48:9: error: ‘Utf8Value’ is not a member of ‘Nan’

         Nan::Utf8Value arg0(info[0]->ToString());